modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -313,6 +313,136 @@ Error GDScriptAnalyzer::resolve_inheritance(GDScriptParser::ClassNode *p_class,
 	class_type.native_type = result.native_type;
 	p_class->set_datatype(class_type);
 
+	#define CHECK_MEMBER_AGAINST_INHERITANCE(name, node) \
```

Comment:
Is there a reason these are macros and not inline functions? Inline functions are generally preferred, see https://github.com/godotengine/godot-proposals/issues/2093#issuecomment-757352474

Reply:
Yeah, I would ask the same.